src/hotspot/cpu/aarch64/gc/z/zGlobals_aarch64.cpp

```diff
@@ -135,6 +140,92 @@
 //  * 63-48 Fixed (16-bits, always zero)
 //
 
+// Default value if probing is not implemented for a certain platform: 128TB
+static const size_t DEFAULT_MAX_ADDRESS_BIT = 47;
+// Minimum value returned, if probing fails: 64GB
+static const size_t MINIMUM_MAX_ADDRESS_BIT = 36;
+
+size_t count_leading_zeros(size_t addr) {
```

Comment:
gcc has builtin count_leading_zeros， __builtin_clzl(unsigned long) , can we use this one?

Reply:
@kuaiwei Sure, I will use the built-in function instead. And I guess it should be contained in "#if defined(TARGET_COMPILER_gcc)", since other compilers might be used. Does dragonwell11 support other compilers besides gcc?